scripts/prebuildify.js

```diff
@@ -30,15 +31,21 @@ function build () {
 
   prebuildify(opts, err => {
     if (err) exit(err)
```

Comment:
Always make prebuilts, but only upload when version actually changed. This will make sure we test that prebuilts can be made. It's a pretty fast step anyway.